backends/memcache.go

```diff
@@ -67,44 +69,108 @@ func (m *Memcache) Execute(cmd string, args ...interface{}) (interface{}, error)
 	return "inside GEt", nil
 }
 
-func (m *Memcache) Delete(keys ...interface{}) int {
-	return 1
+func (m *Memcache) Delete(keys ...interface{}) (int, error) {
```

Comment:
You are right. Unless we to are going to take an action based on type of error, I dont think clients would be interested in knowing about error. Example: err = connection failed => you are trying to reconnect and retry the method.

By returning error from client there will be lot of "if err {}" blocks in clients. My idea was to return negation of expected output in case of errors and panic(with error msg) if an error is critical. Something like no connections in pool is a panic for me. 

Debatable, but I am okay if you find this useful.